package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@thecodecampus/angular-material-theme",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Anheben der Angular Material Mindestversion ist ein Breaking Change. Dann doch lieber
```suggestion
  "version": "4.0.0",
```